app/controllers/openid_application_controller.rb

```diff
@@ -42,8 +42,7 @@ def authenticate_token
       populate_ssoi_oauth_token_payload(profile) unless profile.nil?
       establish_ssoi_session(profile) if @session.nil?
     else
-      profile = fetch_okta_profile(token.identifiers.okta_uid) if profile.nil?
-      populate_ssoi_saml_token_payload(profile) if !profile.nil? && profile.attrs['last_login_type'] == 'ssoi'
+      profile = fetch_okta_profile(token.identifiers.okta_uid) if profile.nil
```

Comment:
I think ? is null safe, do we want to keep that?

Reply:
Oh jeez, that was accidental.  Will get that added back.  Good catch